### ISSUE TITLE
REFL003 when generic GetHashCode

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/ReflectionAnalyzers#80]
```cs
namespace RoslynSandbox
{
    using System.Reflection;

    class Foo
    {
        public MethodInfo Bar<T>() => typeof(T).GetMethod(nameof(this.GetHashCode));
    }
}
```

Should not warn here ^
